src/Spice86.Core/Emulator/Gdb/GdbCustomCommandsHandler.cs

```diff
@@ -345,6 +345,7 @@ public class GdbCustomCommandsHandler {
 
             // Actions for 1 parameter
             if ("refresh".Equals(action)) {
+                Memory memory = _machine.Memory;
```

Comment:
Unused?

Reply:
I'm surprised there is any change at all in gdb related code. I'll revert that.